src/sentry/issues/services/issue/model.py

```diff
@@ -1,6 +1,22 @@
+from datetime import datetime
+
 from sentry.hybridcloud.rpc import RpcModel
 
 
 class RpcGroupShareMetadata(RpcModel):
     title: str
     message: str
+
+
+class RpcExternalIssueGroupMetadata(RpcModel):
```

Comment:
is this info ok to surface? In the issue it says we should only be surfacing the issue link?

Reply:
This is all of the information that is used to render the context pane. I'm not sure if this is correct so I tagged security

Reply:
Updated with issue link and last linked date